modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -121,18 +121,65 @@ In this release, IP restrictions for the connected app are not supported. By def
 
 An error occurs when saving styling changes to the portal if an object has validation rules applied for standard objects that are created during the Salesforce linking process. 
 
-The workaround is to deactivate the validation rule, save the styling changes, then reactive the validation rule: 
-
-. Navigate to *Setup* > *Object Manager*.
-. Select *Validation Rules*.
-. Search for the object with the validation rule.
-. Click *Edit* next to the rule you want to deactivate.
-. Deselect the *Active* checkbox.
-. Click *Save*.
-. Navigate back to API Experience Hub and click *Try again* in the error message.
-. Click *Save* to save your style selections.
-. Repeat steps 1 through 4.
-. Select the *Active* checkbox and click *Save*.
+Custom validation rules can be applied to several Salesforce objects, for example, account and user objects that cause linking and other processes to fail. 
+
+The workaround is to bypass the validation rule for the Salesforce user who performs the linking process. 
+
+NOTE: These instructions pertain to bypassing account object validation rules but can apply to bypassing other object custom validation rules.
+
+. Create a custom permission:
+.. From Salesforce, go to *Setup*, enter *custom permissions* in the Quick Find box, and select *Custom Permissions*.
+.. Click *New* and enter the permission information:
++
+* *Label*
++ 
+Enter *Bypass Account Validation Rule*. 
++
+* *Name* 
++
+Click into the field to populate it with the label name with underscores.
++
+* *Description*
++ 
+Enter a description for the permission.
+.. Click *Save*.
+
+. Create a permission set and add the custom permission:
```

Comment:
This workaround is documented creating a new permission set, however will also possible add the custom permission to an existing permissions set like the one created for minimal permissions initiative. 
The one that is documented [here](https://docs.mulesoft.com/api-experience-hub/assigning-permissions).